src/rubyLsp.ts

```diff
@@ -169,6 +169,14 @@ export class RubyLsp {
 
     await workspace.start();
     this.context.subscriptions.push(workspace);
+
+    // If we successfully activated a workspace, then we can start showing the dependencies tree view. This is necessary
+    // so that we can avoid showing it on non Ruby projects
+    vscode.commands.executeCommand(
+      "setContext",
+      "rubyLsp.showDependenciesTree",
```

Comment:
Can we not make this specific to the dependencies tree? We might want to turn on and off other features depending on whether we activated a Ruby project or not.

What about something like: `rubyLsp.activated`

Reply:
Good point! This event will definitely be generally useful. I switched to `activated`.